### DISCUSSION TITLE
Is this project still being worked on?🤔

### DISCUSSION THREAD

**Discussion by Dav3Plus** [LibreHardwareMonitor/LibreHardwareMonitor#1389] [General]
Last update '31 Dec 23'.  Just wondering👍

**Comment by Dav3Plus** [comment]
Ah, that's good.  Thanks for the quick reply.


Regards....David.
________________________________

**Reply by TheInternetHelpdesk** [reply]
Hopefully it'll inspire more devs to add more features.

**Reply by Davit103** [reply]
I'm trying to get started on it right now.

**Reply by Dav3Plus** [reply]
Excellent 👍

**Comment by Tellinq** [comment]
Technically, that's more of a official version bump artifact release. If you wanted a more reflective state of the project, check the commit history or check out the artifacts that get generated after a merged PR or WIP PR via https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/actions.

As long as there's active communication in PRs between maintainers and offered contributors, this project is alive and well!

**Reply by ebergerly** [reply]
When the vast majority of discussion posts and questions over the last 3 years have gone unanswered, and there is absolutely no documentation whatsoever to describe how these 35,000+ lines of code work, and the code itself has virtually no useful comments, and there's not even an explanation of how someone can add hardware to the database, then good luck finding anyone with nothing better to do than spend months trying to reverse engineer it and produce anything of value. Aside from maybe the hobbyist requests to change colors.... 

Oh wait...someone requested to have the UI colors changed to dark mode about 5 years ago, and nobody could figure out something as simple as changing Form colors and Font colors. In code that's been around for 14 years.  Yeah, the project is alive and well

**Comment by Dav3Plus** [comment]
Yes, lack of documentation /comments a nightmare.  I doubt even the original author will remember how it works by now!


Regards....David.
________________________________
